srcpkgs/juliaup/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'juliaup'
+pkgname=juliaup
+version=1.8.16
+revision=1
+archs="x86_64* i686 aarch64"
+build_style=cargo
+short_desc="Julia installer and version multiplexer"
+maintainer="Bryce Vandegrift <bryce@brycevandegrift.xyz>"
+license="MIT"
+homepage="https://github.com/JuliaLang/juliaup"
+distfiles="https://github.com/JuliaLang/juliaup/archive/refs/tags/v${version}.tar.gz"
+checksum=99b0d62b589c8a330fffaa2252b8e4251124bfd59077f0789a6a456d3b1ae81a
+
+if [ "$XBPS_TARGET_MACHINE" == "x86_64-musl" ]; then
+	# Disable checks for x86_64-musl
```

Comment:
can you explain why they fail?

Reply:
I will enable them and see what happens, they don't fail when I run them natively.

Reply:
I'm not sure why they fail. If I recall correctly, last time I checked, the devs of juliaup disable x86_64-musl checks so I would ask them.